src/components/I18nDropdown.tsx

```diff
@@ -1,42 +1,32 @@
-import { ChangeEvent, FC, useEffect, useRef } from "react";
+import type { FC } from "react";
 import { useTranslation } from "react-i18next";
 import { resources } from "@/i18n/config";
 import { saveSettings } from "@/utils";
+import { Select, SelectItem } from "@nextui-org/react";
 
 const I18nDropdown: FC = () => {
   const { t, i18n } = useTranslation();
-  const selectRef = useRef<HTMLSelectElement>(null);
 
   const langs = Object.keys(resources);
 
-  useEffect(() => {
-    if (selectRef.current) {
-      selectRef.current.value = i18n.language;
-    }
-  }, [i18n]);
-
-  const dropdownHandler = (event: ChangeEvent<HTMLSelectElement>) => {
-    if (event.target.value !== i18n.language) {
-      i18n.changeLanguage(selectRef.current?.value);
-      saveSettings({ lang: i18n.language });
-    }
+  const onSelectionChangeHandler = (event: any) => {
```

Comment:
why `any`?

Reply:
looks like it's their official type? 🙈 
<img width="610" alt="image" src="https://github.com/raspiblitz/raspiblitz-web/assets/1016218/45b6834a-1f4e-41f3-994a-8d7d91614527">